articles/configure-your-rich-text-editors.mdx

```diff
@@ -7,390 +7,153 @@ nav_title: Configure your rich text editors
-
-    Available for `_editables.link`, `_editables.text`, `_editables.image`,`_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-</comp.DataReference>
-
-### Styles and alignment
-
-Add predefined styles in plain CSS for your clients and team members to use in the Visual Editor and Content Editor.
-
-<comp.DataReference>
-  <comp.DataReferenceRow label="styles" type_markdown="String">
-    Enables a drop down menu for editors to style selected text or blocks or text. Styles are the combination of an element and class name. The value for this option is the path (either source or build output) to the CSS file containing the styles.
+_editables___1___:
+  content___2___:
+    bold: true
+    italics: true
+    underline: true
 
-    Available for `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="left" type_markdown="String">
-    Enables a control to left align text by toggling a class name for a block of text. The value is the class name the editor should add to align the text. The styles for this class need to be listed in the `styles` file to take effect outside of the Visual Editor.
+```
+<comp.Annotation number="1">
+  Configure the Content Editor and all editable regions under the `_editables` key anywhere in the [configuration cascade](/documentation/articles/using-the-configuration-cascade/).
+</comp.Annotation>
+<comp.Annotation number="2">
+  Configuration options for the Content Editor are nested keys under `content`.
+</comp.Annotation>
+</comp.MultiCodeBlock>
 
-    Available for `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="center" type_markdown="String">
-    Enables a control to center align text by toggling a class name for a block of text. The value is the class name the editor should add to align the text. The styles for this class need to be listed in the `styles` file to take effect outside of the Visual Editor.
+## Editable regions
 
-    Available for `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="right" type_markdown="String">
-    Enables a control to right align text by toggling a class name for a block of text. The value is the class name the editor should add to align the text. The styles for this class need to be listed in the `styles` file to take effect outside of the Visual Editor.
+There are four types of editable regions in the Visual Editor:
 
-    Available for `_editables.block` and `_editables.content`.
-  </comp.DataReferenceRow>
-  <comp.DataReferenceRow label="justify" type_markdown="String">
-    Enables a control to justify text by toggling a class name for a block of text. The value is the class name the editor should add to justify the text. The styles for this class need to be listed in the `styles` file to take effect outside of the Visual Editor.
+* `text` for simple block editable regions like (e.g. `<p>`)
```

Comment:
I think for our purposes we'd call text inline as well (`p` itself is block, but the user is editing within it so can only create inline content)